app/server.py

```diff
@@ -6,10 +6,11 @@
 from dicttoxml import dicttoxml
 from flask import (Flask, Response, abort, jsonify, make_response,
                    render_template, request)
+
 try:
-    from app.scrapers import feed_gen, scrapers
-except ImportError:
     from scrapers import feed_gen, scrapers
+except Exception as e:
```

Comment:
This seems to be going in the wrong direction.  From PEP8: "_When catching exceptions, mention specific exceptions whenever possible_".  Also, why create an unused variable (__e__)?  Linters will properly complain.